jscomp/others/belt_Int.ml

```diff
@@ -0,0 +1,43 @@
+(* Copyright (C) 2017 Authors of BuckleScript
+ * 
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * In addition to the permissions granted to you by the LGPL, you may combine
+ * or link a "work that uses the Library" with a publicly distributed version
+ * of this file to produce a combined library or application, then distribute
+ * that combined work under the terms of your choosing, with no requirement
+ * to comply with the obligations normally placed on you by section 4 of the
+ * LGPL version 3 (or the corresponding section of a later version of the LGPL
+ * should you choose to use a later version).
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Lesser General Public License for more details.
+ * 
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 59 Temple Place - Suite 330, Boston, MA 02111-1307, USA. *)
+
+(** {!Belt.Int}
+    Utililites for Int
+*)
+
+external toFloat: int -> float = "%identity"
+
+external fromFloat: float -> int = "%intoffloat"
+
+external fromString: string -> (_ [@bs.as 10]) -> int = "parseInt" [@@bs.val]
+
```

Comment:
shall we document its behavior when the input is invalid what will happen, will it raise or coerce to a value？
same applies to Float module

Reply:
I'm wondering what would be the best API there. I guess returning `option(int)` in the case `parseInt` outputs `NaN` would be the best tradeoff. what do you think?